runtime/vm/vmprops.c

```diff
@@ -966,6 +966,11 @@ initializeSystemProperties(J9JavaVM * vm)
 			goto fail;
 		}
 	}
+
+	rc = addSystemProperty(vm, "openjdk.methodhandles", "true", J9SYSPROP_FLAG_WRITEABLE);
```

Comment:
Pls create an appropriately tagged issue to remove this system property after successfully moving to OMH, before a release is created.

Reply:
Created https://github.com/eclipse-openj9/openj9/issues/12811. Embedded it as a comment in the code. Also, there is a mention in the commit message. Similar changes done in https://github.com/adoptium/TKG/pull/198.